packages/@adobe/spectrum-css-temp/components/colorslider/skin.css

```diff
@@ -25,3 +25,13 @@
     margin: 0;
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-ColorSlider {
+    forced-color-adjust: none;
+    --spectrum-colorslider-fill-color-disabled: GrayText;
+    --spectrum-colorslider-border-color-disabled: GrayText;
+    --spectrum-colorslider-border-color: ButtonText;
+    --spectrum-colorarea-border-color: ButtonText;
+  }
```

Comment:
Should the color slider focus outline be ButtonText as opposed to HighLight, so that it is more like a Radio or Checkbox?

Reply:
again there is no color slider focus